.circleci/config.yml

```diff
@@ -72,7 +74,8 @@ jobs:
       - run:
           name: Run package unit tests
           command: |
-            flutter test --coverage
+            flutter test --tags browser --platform chrome
+            flutter test --coverage --exclude-tags browser
```

Comment:
For some reason, adding `--coverage` to `flutter test --tags browser --platform chrome` causes it to produce an error exit code, even though the tests succeed.

Reply:
See flutter/flutter#55437 (comment)

Reply:
See flutter/flutter#55437